core/src/main/java/com/scalar/db/transaction/consensuscommit/CrudHandler.java

```diff
@@ -594,51 +653,25 @@ private Set<AndConditionSet> convertConjunctions(
     return converted;
   }
 
-  private Selection prepareStorageSelection(Selection selection) throws CrudException {
+  private Selection prepareStorageSelection(Selection selection) {
     selection.clearProjections();
-    // Retrieve only the after images columns when including the metadata is disabled, otherwise
-    // retrieve all the columns
-    if (!isIncludeMetadataEnabled) {
-      LinkedHashSet<String> afterImageColumnNames =
-          getTransactionTableMetadata(selection).getAfterImageColumnNames();
-      selection.withProjections(afterImageColumnNames);
-    }
```

Comment:
Stop specifying projections for the after images columns because before images are removed after https://github.com/scalar-labs/scalardb/pull/2787.